### DISCUSSION TITLE
Bug with AlDente 1.28.4

### DISCUSSION THREAD

**Discussion by julienhouyet** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1339] [General]
Hi,

With the last version, discharge or calibrate function don't work. 

Everything indicates that the power supply is switched to battery, but the charger voltage remains high and the battery does not drop.

**Comment by julienhouyet** [comment]
I'm on Sequoia 15.0 - MacBook Pro M3 Pro

<img width="394" alt="Screenshot 2024-09-26 at 06 55 14" src="https://github.com/user-attachments/assets/13512103-57e6-4b43-88d3-49f4dd9bc647">

**Comment by MatthiasKerbl** [comment]
Hi @julienhouyet,

I am sorry to hear you are having issues with AlDente.
Which version of AlDente are you using?

Beste,
Matthias

**Comment by julienhouyet** [comment]
Hi, the 1.28.4 version :)

**Reply by MatthiasKerbl** [reply]
Hi @julienhouyet,

Thank you for the quick reply.
I am currently not able to reproduce your issue. Discharge and Calibration Mode are working without any problems for me.
In general, the charger voltage remains roughly the same even during discharge. However, the charger current should drop to almost 0, as in your screenshot. Furthermore, you can enable Powerflow in settings to visualize the energy flows of your system and to check if the charger is still providing any power. Also, did you wait one hour or more for the battery percentage to drop?

Best,
Matthias

**Comment by julienhouyet** [comment]
Hi,

I have reinstall AlDente et now it's working fine, thanks ! :)
